igor-web/src/main/java/com/netflix/spinnaker/igor/config/BuildMasterProperties.java

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
This is minor, but it seems like we're standardizing on `BuildService` and slowly migrating the jenkins-specific "build master" terminology to "build service".  Based on that it might make more sense to call this `BuildServiceProperties`.

Reply:
Fixed